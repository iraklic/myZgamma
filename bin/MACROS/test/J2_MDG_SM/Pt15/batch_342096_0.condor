000 (342096.000.000) 11/06 14:18:52 Job submitted from host: <131.225.191.129:50550>
...
001 (342096.000.000) 11/06 14:19:17 Job executing on host: <131.225.206.219:53061>
...
006 (342096.000.000) 11/06 14:19:26 Image size of job updated: 10816
...
006 (342096.000.000) 11/06 14:20:27 Image size of job updated: 81320
...
005 (342096.000.000) 11/06 18:01:16 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 03:41:29, Sys 0 00:00:03  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 03:41:29, Sys 0 00:00:03  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	53259  -  Run Bytes Sent By Job
	21846248  -  Run Bytes Received By Job
	53259  -  Total Bytes Sent By Job
	21846248  -  Total Bytes Received By Job
...
