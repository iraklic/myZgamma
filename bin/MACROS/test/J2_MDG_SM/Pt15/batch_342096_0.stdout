
CMSSW on Condor

started at Sun Nov  6 14:19:19 CST 2011

parameter set:
CONDOR_CLUSTER: 342096
CONDOR_PROCESS: 0
CMSSW_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_4_2_0
RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam
PARAMETER_SET: output_Mu_B_S.txt
NUM_EVENTS_PER_JOB: 

Writing final parameter-set: batch_342096_0.py to RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam

run: myMinexam > batch_342096_0.log 2>&1\n

finished at Sun Nov  6 18:01:16 CST 2011
